packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -51,72 +65,126 @@ export default function (plop: NodePlopAPI) {
       },
-    ],
-    actions: [
       {
-        type: 'add',
-        path: currentPath + '/{{project}}/.env',
-        templateFile: '../template/.env.hbs',
-      },
-      {
-        type: 'addMany',
-        destination: currentPath + '/{{project}}/',
-        base: '../template',
-        templateFiles: [
-          '../template/src/**',
-          '../template/scripts/*',
-          '../template/.env.example',
-          '../template/tsconfig.json',
-          '../template/stencil.config.ts',
-          '../template/netlify.toml',
-          '../template/README.md',
-          '../template/start-netlify.mjs',
-        ],
-      },
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/.gitignore',
-        templateFile: '../template/.gitignore.hbs',
-      },
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/package.json',
-        templateFile: '../template/package.json.hbs',
-      },
-      function installPackagesPrompt() {
-        return 'Installing packages...';
+        type: 'input',
+        name: 'page-id',
+        default: '',
+        message:
+          '(Optional) The hosted search page ID, e.g. "7944ff4a-9943-4999-a3f6-3e81a7f6fb0a".',
       },
-      function installPackages(answers) {
-        return new Promise((resolve, reject) => {
-          const {project} = answers as PromptsAnswers;
-          const process = spawn(getPackageManager(), ['install'], {
-            stdio: 'ignore',
-            cwd: `${currentPath}/${project}/`,
-          });
+    ],
+    actions: function () {
+      return [
+        function downloadSearchPagePrompt(data) {
+          const plopData = data as PlopData;
+          if (plopData['page-id'] !== '') {
+            return 'Downloading Hosted Search Page';
+          }
+
+          return '';
+        },
+        async function downloadSearchPage(data) {
+          const plopData = data as PlopData;
+          if (plopData['page-id'] === '') {
+            plopData.page = defaultPageManifest;
+            return '';
+          }
 
-          process.on('close', (code) => {
-            if (code === 0) {
-              resolve('Installation complete');
-            } else {
-              reject(`Installation exited with ${code}`);
-            }
+          try {
+            plopData.page = await fetchPageManifest(
+              // TODO: replace
+              'http://localhost:8222', // plopData['platform-url'],
+              plopData['org-id'],
+              plopData['page-id'],
+              plopData['api-key']
+            );
+
+            return `Hosted search page "${plopData.page.config.title}" downloaded`;
+          } catch (error) {
+            throw new Error(
+              `There was an error downloading search page with id "${plopData['page-id']}" from the organization "${plopData['org-id']}"`
```

Comment:
If the error is that we don't have the privileges to download search pages, I think we should throw a more specific error.

Reply:
Will handle in platform client